lib/atat-iam-stack.ts

```diff
@@ -117,7 +117,7 @@ export class AtatIamStack extends cdk.Stack {
         // troubleshoot
         new iam.PolicyStatement({
           effect: iam.Effect.ALLOW,
-          actions: ["sqs:Get*", "sqs:List*", "sqs:ReceiveMessage"],
+          actions: ["sqs:Get*", "sqs:List*", "sqs:ReceiveMessage", "sqs:PurgeQueue"],
```

Comment:
Thank you, I will use this with care 🙂. The `sqs:PurgeQueue` appears to wipe out the whole queue, which is probably fine, but I was wondering if `sqs:DeleteMessage` might be a less destructive permission since you can select individual messages (assuming other ones shouldn't be touched)?

Reply:
`PurgeQueue` is a button in the Console which helps :) sometimes the messages get stuck "in-flight" between the failures and `PurgeQueue` seems to be a little better at dealing with that.